### PR TITLE
Bump to 0.6.0.dev0

### PR DESCRIPTION
**Pull request by anton-l**

### CODE REVIEW

File: src/diffusers/schedulers/scheduling_ddim.py [hunk 1]
```diff
@@ -121,13 +121,6 @@ def __init__(
         steps_offset: int = 0,
         **kwargs,
     ):
-        deprecate(
-            "tensor_format",
-            "0.6.0",
-            "If you're running your code in PyTorch, you can safely remove this argument.",
-            take_from=kwargs,
-        )
-
```
Comment (anton-l) [inline src/diffusers/schedulers/scheduling_ddim.py:130]: Removing these deprecations now, as they're fairly harmless.

File: src/diffusers/schedulers/scheduling_utils.py [hunk 2]
```diff
@@ -45,7 +45,7 @@ class SchedulerMixin:
     def set_format(self, tensor_format="pt"):
         deprecate(
             "set_format",
-            "0.6.0",
+            "0.7.0",
             "If you're running your code in PyTorch, you can safely remove this function as the schedulers are always"
             " in Pytorch",
         )
```
Comment (anton-l) [inline src/diffusers/schedulers/scheduling_utils.py:51]: We can either bump or deprecate this function, but I'm ok with keeping it a bit longer

File: src/diffusers/utils/outputs.py [hunk 3]
```diff
@@ -86,9 +85,6 @@ def update(self, *args, **kwargs):
     def __getitem__(self, k):
         if isinstance(k, str):
             inner_dict = {k: v for (k, v) in self.items()}
-            if self.__class__.__name__ in ["StableDiffusionPipelineOutput", "ImagePipelineOutput"] and k == "sample":
-                deprecate("samples", "0.6.0", "Please use `.images` or `'images'` instead.")
-                return inner_dict["images"]
```
Comment (anton-l) [inline src/diffusers/utils/outputs.py:91]: Feels safe to deprecate this now

**Comment by patrickvonplaten** [comment]
Might have some merge conflicts here: https://github.com/huggingface/diffusers/pull/833

**Comment by anton-l** [comment]
Resolved :+1:

File: src/diffusers/schedulers/scheduling_ddim.py (hunk 1 above)
Comment (patrickvonplaten) [reply src/diffusers/schedulers/scheduling_ddim.py:130]: `**kwargs` has to be removed as well as it's now left unchecked and it was added only for backwards breaking comp which we're breaking now

File: src/diffusers/schedulers/scheduling_ddpm.py [hunk 4]
```diff
@@ -114,13 +114,6 @@ def __init__(
         clip_sample: bool = True,
         **kwargs,
```
Comment (patrickvonplaten) [inline src/diffusers/schedulers/scheduling_ddpm.py:115]:
```suggestion
```

File: src/diffusers/schedulers/scheduling_ddim.py [hunk 5]
```diff
@@ -121,13 +121,6 @@ def __init__(
         steps_offset: int = 0,
         **kwargs,
```
Comment (patrickvonplaten) [inline src/diffusers/schedulers/scheduling_ddim.py:122]:
```suggestion
```

File: src/diffusers/schedulers/scheduling_karras_ve.py [hunk 6]
```diff
@@ -88,13 +88,6 @@ def __init__(
         s_max: float = 50,
         **kwargs,
```
Comment (patrickvonplaten) [inline src/diffusers/schedulers/scheduling_karras_ve.py:89]:
```suggestion
```

File: src/diffusers/schedulers/scheduling_lms_discrete.py [hunk 7]
```diff
@@ -76,13 +76,6 @@ def __init__(
         trained_betas: Optional[np.ndarray] = None,
         **kwargs,
```
Comment (patrickvonplaten) [inline src/diffusers/schedulers/scheduling_lms_discrete.py:77]:
```suggestion
```

File: src/diffusers/schedulers/scheduling_pndm.py [hunk 8]
```diff
@@ -102,13 +102,6 @@ def __init__(
         steps_offset: int = 0,
         **kwargs,
```
Comment (patrickvonplaten) [inline src/diffusers/schedulers/scheduling_pndm.py:103]:
```suggestion
```

File: src/diffusers/schedulers/scheduling_sde_vp.py [hunk 9]
```diff
@@ -41,12 +40,6 @@ class ScoreSdeVpScheduler(SchedulerMixin, ConfigMixin):
 
     @register_to_config
     def __init__(self, num_train_timesteps=2000, beta_min=0.1, beta_max=20, sampling_eps=1e-3, **kwargs):
```
Comment (patrickvonplaten) [inline src/diffusers/schedulers/scheduling_sde_vp.py:42]:
```suggestion
    def __init__(self, num_train_timesteps=2000, beta_min=0.1, beta_max=20, sampling_eps=1e-3):
```

**Review by patrickvonplaten — APPROVED** [review]
Thanks a lot for making the changes. We can and should remove the `**kwargs` arg from init now IMO.

**Pull request merged**